app/javascript/alchemy_admin/utils/spinner.js

```diff
@@ -0,0 +1,11 @@
+export function appendSpinner(element, size = "small") {
```

Comment:
I think it makes sense to have this file in `alchemy_admin/spinner.js`, not in `utils`, because we will convert the spinner as well soon.

Reply:
I moved the `appendSpinner` and `removeSpinner` into the web component. We can extract them, if we use these functions anywhere else.